crates/symbolicator/src/services/download/mod.rs

```diff
@@ -60,15 +63,36 @@ pub enum DownloadError {
 }
 
 impl DownloadError {
+    /// This produces a user-facing string representation of a download error if it is a variant
+    /// that needs to be stored as a `CacheStatus::CacheSpecificError` entry in the download cache.
```

Comment:
```suggestion
    /// that needs to be stored as a [`CacheStatus::CacheSpecificError`] entry in the download cache.
```

This turns these into links in rustdoc btw.